app/styles/buttons.ts

```diff
@@ -106,3 +112,10 @@ export const largeTransparent: ViewStyle = {
   ...large,
   ...transparent,
 };
+
+export const pill: ViewStyle = {
+  borderRadius: 100,
```

Comment:
I'm assuming this value is very specific to this component and that's why it's not in a variable

Reply:
It may make sense to either create a variable called `borderRadiusMax` and set it to a big number like `1000` or, if we can, just use `100%` here. Not sure if that works in RN world though. @jeffstolz The goal here is to get fully rounded corners.

Reply:
Yeah I'd use `100%` here then - I think it's much more explicit about its purpose and more fail-safe

Reply:
totally agree, will tackle in another pr as to not have to go through all of ci again.